lib/Symfony/SearchBundle/DependencyInjection/Configuration.php

```diff
@@ -38,7 +39,7 @@ public function getConfigTreeBuilder()
             ->children()
                 ->arrayNode('processor')
                     ->info('SearchProcessor configuration')
-                    ->{class_exists(Psr7SearchProcessor::class) && interface_exists(HttpFoundationFactoryInterface::class) ? 'canBeDisabled' : 'canBeEnabled'}()
+                    ->{class_exists(Psr7SearchProcessor::class) && class_exists(HttpFoundationFactoryInterface::class) ? 'canBeDisabled' : 'canBeEnabled'}()
```

Comment:
This needs to be reverted. Properly a bad rebase. I have no idea why this even worked in the first place 😳

Reply:
Yeah, I did not do that, might have been CS fixer.